paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -208,8 +208,8 @@ class Paparazzi @JvmOverloads constructor(
   }
 
   @JvmOverloads
-  fun snapshot(view: View, name: String? = null) {
-    takeSnapshots(view, name, 0, -1, 1)
+  fun snapshot(view: View, name: String? = null, timeNanos: Long = 0L) {
```

Comment:
I didn’t add an overload for compose, and I probably should. Does the timeNanos argument slot in before the Composable? And if so, do I need a manual overload?

Reply:
My instinct says yes, should support Compose but maybe in a separate PR because I don't think it'll "just work" without some extra work.

Yes, to slotting before composable.

Why manual overload?  Cuz major/minor compatibility?

Reply:
Yep. I think that in order to keep the lambda argument last we can’t have defaulted params that precede it?